beacon-chain/powchain/service.go

```diff
@@ -212,6 +213,14 @@ func NewService(ctx context.Context, opts ...Option) (*Service, error) {
 		}
 	}
 
+	if err := s.connectToPowChain(); err != nil {
```

Comment:
We now connect to the powchain on initialization, as this is required for the merge